src/pages/account/logins/index.tsx

```diff
@@ -20,7 +24,16 @@ type LoginsProps = {
 };
 
 const LoginsPage: React.FC<LoginsProps> = ({ sortedIdentities }) => {
-  return <Logins sortedIdentities={sortedIdentities} />;
+  return (
+    <Container>
+      <Head>
+        <title>Connect Logins</title>
+      </Head>
```

Comment:
Can you remove the head component? In my PR, I added it right into the `_document` component.

Reply:
OK fixed e68cb79